pkg/network/config.go

```diff
@@ -1,4 +1,88 @@
 package network
 
-// CNIConfigDir is the path where CNI config files exist
-const CNIConfigDir = "/etc/cni/net.d"
+import (
+	"encoding/json"
+	"net"
+)
+
+// TODO once the libpod.conf file stuff is worked out, this should be modified
+// to honor defines in the libpod.conf as well as overrides?
+
+const (
+	// CNIConfigDir is the path where CNI config files exist
+	CNIConfigDir = "/etc/cni/net.d"
+	// CNIDeviceName is the default network device name and in
+	// reality should have an int appended to it (cni-podman4)
+	CNIDeviceName = "cni-podman"
+)
+
+// GetDefaultPodmanNetwork outputs the default network for podman
+func GetDefaultPodmanNetwork() (*net.IPNet, error) {
+	_, n, err := net.ParseCIDR("10.88.1.0/24")
```

Comment:
We gotta make this configurable somewhere - `containers.conf`, `libpod.conf`, somewhere

Reply:
agree but I think that is basically hanging on the whole conf file discussion.  for now, id like to do this and then return to it.